launch-or-update-local.sh

```diff
@@ -53,9 +61,15 @@ if [[ "$context" != "docker-desktop" ]]; then
   exit 1
 fi
 
+if [ "$latest" != "false" ]
+  then 
+    echo "Getting Latest Chart Versions..."$'\n'
+    ./get-latest-charts.sh
+fi
+
```

Comment:
This is just a personal preference, so please, fee free to ignore.

I'm a big fan of the `if ...; then` syntax rather than
```
if ...
   then
```
syntax.

I feel like splitting the then keyword on another line bloats the statement conditional and makes it harder to read.
This is what I mean, here's the same code side by side
```
if [ "$latest" != "false" ]
  then 
    echo "Getting Latest Chart Versions..."$'\n'
    ./get-latest-charts.sh
fi
```
```
if [ "$latest" != "false" ]; then 
  echo "Getting Latest Chart Versions..."$'\n'
  ./get-latest-charts.sh
fi
```

But as I said, this is just a personal preference in terms of the syntax that doesn't reflect on the functionality.

Reply:
Updated to match other uses of if... then...